BLSQ/iaso
# IA-3364 user history front end

Explain what problem this PR is resolving

Related JIRA tickets : IA-3364

## Self proofreading checklist


## Doc


## Changes

- Add sub menu to "users" menu entry.
- Use `USERS_ADMIN" permission to control access
- List view:
    - Call `userlogs` api
    - Populate table
    - Filters:
        - users
        - modification date
        - location: filters on past and new org units
- Details view:
    - Use table `subComponent` to expand the view under the current row
    - call `logs` api to get data
    - Duplicate all logs related components and adapt them for users   

## How to test

- Go to users > management
- create a user
- update a user
- update multiple users by selecting them via the table. IMPORTANT: you need an active worker for this part
- create multiple users via csv

Go to users > history:
- All creations and updates should appear in the table
- Test out all the filters
- Expand logs to compare before and after views.



## Print screen / video

Uploading Screen Recording 2024-09-09 at 17.08.00.mov…



## Notes

Merges into https://github.com/BLSQ/iaso/pull/1614
